serverless/functions/serverless-developing-go-functions.adoc

```diff
@@ -0,0 +1,27 @@
+include::modules/serverless-document-attributes.adoc[]
+[id="serverless-developing-go-functions"]
+= Developing Golang functions
+:context: serverless-developing-go-functions
+include::modules/common-attributes.adoc[]
+
+toc::[]
+
+:FeatureName: {FunctionsProductName}
+include::modules/technology-preview.adoc[leveloffset=+2]
+
+After you have xref:../../serverless/functions/serverless-functions-getting-started.adoc#serverless-functions-getting-started[created a Golang function project], you can modify the template files provided to add business logic to your function.
+
+[id="prerequisites_serverless-developing-go-functions"]
+== Prerequisites
+
+* Before you can develop functions, you must complete the set up steps in xref:../../serverless/functions/serverless-functions-setup.adoc#serverless-functions-setup[Setting up {FunctionsProductName}].
+
+include::modules/serverless-go-template.adoc[leveloffset=+1]
+include::modules/serverless-invoking-go-functions.adoc[leveloffset=+1]
+include::modules/serverless-go-function-return-values.adoc[leveloffset=+1]
+include::modules/serverless-testing-go-functions.adoc[leveloffset=+1]
+
+[id="next-steps_serverless-developing-go-functions"]
+== Next steps
+
+* xref:../../serverless/functions/serverless-functions-getting-started.adoc#serverless-functions-getting-started[Build and deploy a function].
```

Comment:
Just like the xref at the beginning of the assembly, this generic link to xref:../../serverless/functions/serverless-functions-getting-started.adoc#serverless-functions-getting-started threw me off. It just lands at the top of the "Getting started" assembly.

It's the same one used at the top of the assembly, but with a different link text. And the one at the top of the assembly makes it seem like it's something that needs to be done first, but now it's also added as a next step. So not sure if there's something more specific that you can link to in each of these scenarios to make it clearer?